### DISCUSSION TITLE
If my skill is above average, is it easier to go down than up?

### DISCUSSION THREAD

**Discussion by jauggy** [OpenDebates/openskill.py#131] [Q&A] [answered]
So from what I understand is that the distribution of OS is normal. So there's more people around the mean. Does that mean if I'm above average it's easier to go down than up in terms of skill?

**Comment by vivekjoshy** [comment, marked as answer]
That depends on the skill level of your opponent and the model in question. For instance, the Plackett-Luce model's distribution isn't technically a normal one even though the relationship between $\mu$ and $\sigma$ parameters for an individual skill as implemented in Weng-Lin are normal. The Plackett-Luce distribution isn't even technically a univariate distribution like the Gaussian.

If you're looking for intuition, the traditional comparison to Elo can be helpful since it's proved in the paper Elo is a special case of the models.

TLDR: No. It depends on the distribution of skilled players. More skilled players than you mean, higher chance of your skill going down. When you go against expectations, your skill increases, and confidence improves.
